src/components/AnacondaWizard.jsx

```diff
@@ -103,159 +109,168 @@ export const AnacondaWizard = ({ dispatch, storageData, localizationData, runtim
             }
         }
     }, [localizationData]);
-    const stepsOrder = [
-        {
-            component: InstallationLanguage,
-            data: { dispatch, languages: localizationData.languages, language: localizationData.language, commonLocales: localizationData.commonLocales },
-            ...getInstallationLanguageProps({ isBootIso, osRelease })
-        },
-        {
-            component: InstallationMethod,
-            data: {
-                deviceData: storageData.devices,
-                diskSelection: storageData.diskSelection,
-                dispatch,
-                storageScenarioId,
-                setStorageScenarioId: (scenarioId) => {
-                    window.sessionStorage.setItem("storage-scenario-id", scenarioId);
-                    setStorageScenarioId(scenarioId);
-                }
+    const stepsOrder = useMemo(() => {
```

Comment:
What happens if stepsOrder is not memoized? Did you try without?

Reply:
This is from the part where I tried to disable the next steps. I'm going to go through the code again to see if there's anything like that somewhere.